packages/status-js/src/consts/peers.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * source: @see https://fleets.status.im
+ */
+
+// note!: users may experience additional latency due to cross-regional connection
+// todo: use "dynamic" discovery protocol instead
+// todo?: use a regional map together with an environment variable for the peer selection (e.g. `VERCEL_REGION`, but probably limited to  Serverless Functions)
+export const peers = {
+  production: [
+    // Nim
```

Comment:
note: only one prod node worked for me, didn't spend time investigating atm. Probably due to the Relay connection cap, and a reason to switch to Filter and Light Push for now https://github.com/status-im/status-web/pull/301.